packages/web/app/src/pages/target-settings.tsx

```diff
@@ -488,6 +488,9 @@ const ConditionalBreakingChanges = (props: {
   const isEnabled = settings?.enabled || false;
   const possibleTargets = targetSettings.data?.targets.nodes;
   const { toast } = useToast();
+  const retentionInDays =
+    targetSettings.data?.organization?.organization?.rateLimit.retentionInDays ?? 30;
+  const defaultDays = retentionInDays >= 30 ? 30 : 7;
```

Comment:
can we make `30` and `7` constants in outside the component? or maybe even put them as a shared constant